packages/help-center/src/components/help-center-search-results.tsx

```diff
@@ -224,6 +229,20 @@ function HelpSearchResults( {
 			return <Icon icon={ pageIcon } />;
 		};
 
+		const DeveloperBadge = () => {
+			return (
+				<div className="help-center-search-results__content">
+					{ isResultFromDeveloperWordpress( result.link ) && (
+						<Badge type="info" className="help-center-search-results__badge">
+							{ /* translators: Dev is an acronym of Developers */ }
+							{ __( 'Dev', __i18n_text_domain__ ) }
+						</Badge>
+					) }
+					<span>{ preventWidows( decodeEntities( title ) ) }</span>
```

Comment:
Nit: not sure it's necessary to have this as `<DeveloperBadge />`  - might be a little confusing that the title is included here? Although I recognise that you need to wrap it in a `div`, so might be necessary!

Reply:
I like it because it is a wrapper.